modes/passenger-services.md

```diff
@@ -0,0 +1,204 @@
+| `stopped`                    | `available`       | N/A          | `provider_cancellation` | The provider has canceled the trip while the vehicle is waiting for a passenger, or dropping them off |
+| `stopped`                    | `available`       | N/A          | `trip_end`               | The trip has been successfully completed                                                                         |
+| `stopped`                    | `on_trip`         | `on_trip`    | `trip_resume`            | Resume a trip that was previously stopped (e.g. picking up a friend to go to the airport with)                   |
+| `stopped`                    | `on_trip`         | `on_trip`    | `trip_start`             | Start a trip                                                                                                     |
+| `stopped`                    | `unknown`         | N/A          | `comms_lost`             | The vehicle has went out of comms while stopped                                                                  |
+| `unknown`                    | `available`       | N/A          | `comms_restored`         | The vehicle has come back into comms while available for-hire                                                    |
+| `unknown`                    | `elsewhere`       | N/A          | `comms_restored`         | The vehicle has come back into comms while outside of jurisdictional boundaries                                  |
+| `unknown`                    | `non_operational` | N/A          | `comms_restored`         | The vehicle has come back into comms while not operating commercially                                            |
+| `unknown`                    | `on_trip`         | `on_trip`    | `comms_restored`         | The vehicle has come back into comms while on a trip                                                             |
+| `unknown`                    | `removed`         | N/A          | `comms_restored`         | The vehicle has come back into comms while removed                                                               |
+| `unknown`                    | `reserved`        | `reserved`   | `comms_restored`         | The vehicle has come back into comms while reserved by a passenger                                               |
+| `unknown`                    | `stopped`         | `stopped`    | `comms_restored`         | The vehicle has come back into comms while stopped                                                               |
+
+### State Machine Diagram
+
+The *Passenger Services State Machine Diagram* shows how the `vehicle_state` and `event_type` relate to each other and how taxi vehicles can transition between states.
+
+![Passenger Services State Machine Diagram](passenger-services-state-machine-diagram.svg)
```

Comment:
@avatarneil @marie-x do you have a google doc or SVG source file for this? I'd like to put it into an editable format and link to it here, like we do for micromobility.

Reply:
We built the diagram in Figma, I could see about sharing out an editable version of it. Is the SVG already in here unworkable when loaded into a vector graphics editor?

Reply:
https://www.figma.com/file/T4YW9OxwdePvKhy3j9UD8k/Untitled?node-id=1%3A10

Please let me know if this works @schnuerle!

Reply:
Thank you! Added an EPS version to the spec here. https://github.com/openmobilityfoundation/mobility-data-specification/pull/763/commits/2996a6375f3bd03ea3f4e7105f4fd98c10967119